test/dotcom_web/channels/predictions_channel_test.exs

```diff
@@ -2,9 +2,10 @@ defmodule DotcomWeb.PredictionsChannelTest do
   use DotcomWeb.ChannelCase, async: false
 
   import Mox
+  import Dotcom.Utils.DateTime, only: [now: 0]
```

Comment:
Should credo flag this as not being in alphabetical order?

Reply:
:( credo isn't smart enough to alphabetize sadly, but thanks for calling it out here